tests/translator/output/api_request_model.json

```diff
@@ -29,30 +29,30 @@
           ]
         }
       }
-    }, 
+    },
     "HtmlApiProdStage": {
-      "Type": "AWS::ApiGateway::Stage", 
+      "Type": "AWS::ApiGateway::Stage",
       "Properties": {
         "DeploymentId": {
-          "Ref": "HtmlApiDeploymentdd6198efe6"
-        }, 
+          "Ref": "HtmlApiDeploymentcb065c609e"
```

Comment:
I don't think it's because of py2 vs py3. Rather, since there's a new path added in the input template, I believe the hash change is due to swagger change.

Reply:
@mndeveci @hawflau is correct. All changes here are due to adding new paths to an API Gateway and therefore changing the hashes. I thought about maybe splitting everything into new tests but figured this was easier on me and that there are no code changes, so the hashes and things changing were predictable and expected.

Reply:
Thanks for the explanation @jfuss & @hawflau !